src/exam/Exam.jsx

```diff
@@ -70,6 +77,7 @@ const Exam = ({
     // this makes sure useEffect gets called only one time after the exam has been fetched
     // we can't leave this empty since initially exam is just an empty object, so
     // API calls above would not get triggered
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
[question] I noticed this in a couple of places. Could you clarify the need for this / if this will eventually be unnecessary?

Reply:
This lint rule is meant to try and prevent developers from forgetting dependencies in the use effect. Because I didn't change anything within the use effect, I'm planning to leave this as is for now!

Reply:
Should it be removed?

I'm wondering if it's complaining because you're using `dispatch` in the `useEffect` hook, so `dispatch` has to be added to the dependency array.

Reply:
Oh, fair point! I will try adding that in and see what happens.